cmake/modules/CheckYasm.cmake

```diff
@@ -22,21 +22,21 @@ macro(check_yasm_support _object_format _support_x64 _better_result)
           ${CMAKE_SOURCE_DIR}/src/isa-l/include/
           ${CMAKE_SOURCE_DIR}/src/isa-l/erasure_code/gf_vect_dot_prod_avx2.asm
           -o /dev/null
-          RESULT_VARIABLE rc
+          RESULT_VARIABLE not_support_avx2
```

Comment:
i'd avoid using `not_support_avx2` here. `NOT not_support_avx2` is more difficult to read.

Reply:
I'll keep "rc" there since I don't have better idea.